specs/ledger/latex/update-mechanism.tex

```diff
@@ -138,8 +138,9 @@ \subsection{Update proposals}
   \label{fig:defs:update-proposals}
 \end{figure}
 
-The set of protocol parameters ($\ProtPm$) is assumed to contain the following keys, which
-correspond with fields of the current `BlockVersionData` structure:
+The set of protocol parameters ($\ProtPm$) is assumed to contain the following
+keys, some of which correspond with fields of the current `BlockVersionData`
```

Comment:
Changed the name of `BlockVersionData` to `ProtocolParameters` :)

Reply:
Oh, thanks for that! However we're referring here to the `cardano-sl` implementation. I don't know whether it is confusing to talk about it here. Maybe we want to remove all references to it. But on the other hand, we're modeling things the way we do because of the legacy system. So I don't know :man_shrugging: